railties/test/application/configuration_test.rb

```diff
@@ -1293,6 +1293,32 @@ def index
       assert_equal false, ActionView::Resolver.caching?
     end
 
+    test "ActionController::Base.raise_on_open_redirects is true by default for new apps" do
+      app "development"
+
+      assert_equal true, ActionController::Base.raise_on_open_redirects
+    end
+
+    test "ActionController::Base.raise_on_open_redirects is false by default for upgraded apps" do
+      remove_from_config '.*config\.load_defaults.*\n'
+      add_to_config 'config.load_defaults "6.1"'
+      app "development"
+
+      assert_equal false, ActionController::Base.raise_on_open_redirects
+    end
+
+    test "ActionController::Base.raise_on_open_redirects can be configured in the new framework defaults" do
+      remove_from_config '.*config\.load_defaults.*\n'
+
+      app_file "config/initializers/new_framework_defaults_6_2.rb", <<-RUBY
```

Comment:
```suggestion
      app_file "config/initializers/new_framework_defaults_7_0.rb", <<-RUBY
```

doesn't *really* matter, don't let this block merging.